public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -743,25 +743,27 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
     return this._serializer.getDashboardChangesFromScene(this, { saveTimeRange, saveVariables, saveRefresh });
   }
 
-  /**
-   * Provisioned dashboards helpers
-   */
-  getRepoName() {
-    return this.state.meta.k8s?.annotations?.[AnnoKeyManagerIdentity];
+  isManaged() {
+    return Boolean(this.state.meta.k8s?.annotations?.[AnnoKeyManagerKind]);
```

Comment:
I'd also add a property here to get `dashboard.state.meta.k8s?.annotations?.[AnnoKeyManagerKind]`, so that's easier, e.g., `getManagerKind`.